### PR TITLE
migrate to FloatingMenu, almost

### PR DESCRIPTION
**Pull request by songgao**
On mobile we are still using ModalLessPopup for the download popup, which I think is the last one. There are some changes required around here from M.2 QA, so the whole thing may need to change. I'll do them in a separate PR.

~Opened against https://github.com/keybase/client/pull/13144 since this touches stuff from that. As a result CircleCI will also fail until rebase.~

### CODE REVIEW

File: shared/fs/sortbar/sortbar.js [hunk 1]
```diff
@@ -62,4 +82,48 @@ const SortBar = (props: SortBarProps) => {
   )
 }
 
-export default SortBar
+// TODO Styles.styleSheetCreate doesn't work with icon
```
Comment (buoyad) [inline shared/fs/sortbar/sortbar.js]: same here

File: shared/fs/sortbar/sortbar.js [hunk 2]
```diff
@@ -1,59 +1,79 @@
...
+      <Box style={styles.sortBar}>
+        <ClickableBox onClick={props.toggleShowingMenu} style={styles.sortSetting}>
+          <Box style={styles.iconBox}>
+            <Icon type={sortSettingIconType} style={styles.icon} fontSize={11} />
           </Box>
           <Text type="BodySmallSemibold">{sortSettingText}</Text>
         </ClickableBox>
+        <FloatingMenu
+          containerStyle={styles.popup}
+          attachTo={props.attachmentRef}
```
Comment (buoyad) [inline shared/fs/sortbar/sortbar.js]: you're missing a `setAttachmentRef`. this isn't attached to anything

File: shared/fs/sortbar/sortbar.js [hunk 3]
```diff
@@ -62,4 +82,48 @@ const SortBar = (props: SortBarProps) => {
   )
 }
 
-export default SortBar
+// TODO Styles.styleSheetCreate doesn't work with icon
+const styles = {
+  popup: Styles.platformStyles({
```
Comment (buoyad) [inline shared/fs/sortbar/sortbar.js]: This shouldn't be needed

**Comment by songgao** [comment]
@buoyad this is ready for another look; thanks!

File: shared/fs/common/open-in-fileui-container.desktop.js [hunk 4]
```diff
@@ -3,13 +3,11 @@ import * as FsGen from '../../actions/fs-gen'
...
 import {compose, connect, setDisplayName, type TypedState, type Dispatch} from '../../util/container'
-import {type OpenInFileUIProps} from './open-in-fileui-container'
+import {type OpenInFileUIProps as OwnProps} from './open-in-fileui-container'
 import OpenInFileUI from './open-in-fileui.desktop'
 
-type OwnProps = OpenInFileUIProps
-
 const mapStateToProps = (state: TypedState, {path}: OwnProps) => ({
-  kbfsEnabled: Constants.kbfsEnabled(state),
+  kbfsEnabled: false && Constants.kbfsEnabled(state),
```
Comment (buoyad) [inline shared/fs/common/open-in-fileui-container.desktop.js]: ❗️

Comment (songgao) [reply shared/fs/common/open-in-fileui-container.desktop.js]: whoops

**Pull request merged**